esp/esp/program/modules/handlers/onsiteclasslist.py

```diff
@@ -448,7 +448,7 @@ def classList_base(self, request, tl, one, two, module, extra, prog, options=Non
         else:
             context['use_categories'] = True
         
-        return render_to_response(self.baseDir()+'classlist.html', request, context)
+        return render_to_response(self.baseDir()+template_name, request, context)
```

Comment:
Does this mess up `classlist_public`, which appears to set the previously unused `template_name` argument (to something that's not `classlist.html`)?  (Or does it fix it?)